wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,331 @@
+
+- `osu! Alumni`
+- `osu! Champion`
+
+---
+
+`osu!` should not end the sentence. If it does however, use a full stop (`.`) (or question mark (`?`), if appropriate) to get `osu!.` (or `osu!?`).
+
+`osu!` must not be followed by an exclamation mark (`!`) (i.e. `osu!!`). It must be rewritten to ensure that this will not occur.
+
+## Terminology
+
+These words must be spelt as follows (spacing must match):
+
+- `approach circle`
+- `game mode` (or `mode`)
+- `game mod` (or `mod`)
+- `gameplay`
+- `hit burst`
+- `hit circle`
+- `hitsound`
+- `in-game`
+- `playstyle`
+- `slider tick`
+- `slider ball`
+- `slider path`
+
+---
+
+Some words have variants. Their preferred spelling must be used and is as follows:
+
+- `beatmap` instead of `map`.
+- `creator` instead of `beatmapper` or `mapper`.
+- `mapped` instead of `beatmapped`.
+<!-- https://github.com/ppy/osu-wiki/pull/561#discussion_r132090032 -->
+- `BN` or `Beatmap Nominators` when referring to the *Beatmap Nominators*.
+- `sign in` instead of `log in`, unless the name of a button or link says otherwise.
+- `sign out` instead of `log out`, unless the name of a button or link says otherwise.
+- `register` instead of `sign up`, unless the name of a button or link says otherwise.
+
+### Tournament articles
+
+These words must be spelt as follows (spacing must match):
+
+- `NoMods`, `NoMod`, or `NM`
+- `FreeMods`, `FreeMod`, or `FM`
+- `Tiebreaker`
+
+## Abbreviations, acronyms, and initialisms
+
+Abbreviations, acronyms, and initialisms must have their meaning written out upon their first occurrence. Other occurrences are optional, but done only when necessary. For example:
+
+```markdown
+The NC (Nightcore) mod is similar to the DT (Double Time) mod because both NC and DT increase the speed of the music by 50%. However, NC will change the pitch of the music and adds a clap and finish to the beat.
+```
+
+Abbreviations, acronyms, and initialisms must be capitalised, with exception. For example:
+
+- `CS` for `Circle Size`
+- `AR` for `Approach Rate`
+- `DT` for `Double Time`
+- `SBS` for `Storyboard Scripting`
+
+---
+
+Use `e.g.` for "for example" and `i.e.` for "that is".
+
+---
+
+Abbreviations, acronyms, and initialisms should not be pluralised. For example:
+
+```markdown
+Avoid:  BNs can nominate your beatmapset.
+
+Prefer: Beatmap Nominators (BN) can nominate your beatmapset.
+```
+
+## Perspective
+
+`player`, `user`, `skinner`, `storyboarder`, and `creator` should be used when referring to the reader or another person. `they`, `them`, `their`, and `theirs` may be used when needed.
+
+`you` and `your` should be avoided. `I` must not be used. `we`, `he` or `his`, and `she` or `her` must not be used (see above for other terms).
+
+---
+
+[Articles with informal registers](#language-register) may ignore this section; however, `I` in paragraphs should be avoided.
+
+## Grammar and syntax
+
+Articles should prefer using simpler grammar and syntax.
```

Comment:
We should specifically mention which style of punctuation ([American vs. British](https://www.thepunctuationguide.com/british-versus-american-style.html)) should be used for articles. As well as any specific changes to those styles that we make.